translation/pt-br/patterns/base-documentation.md

```diff
@@ -32,7 +32,7 @@ Abordar a necessidade de documentação mais clara para novos contribuidores. O
 
 ### README.md
 
-Se ainda não existir, crie um arquivo `README.md para o seu projeto. Ele deve conter:
+Se ainda não existir, crie um arquivo `README.md` para o seu projeto. Ele deve conter:
```

Comment:
I know that this is a small fix for a typo, however it is on a different language, so properly speaking this PR is only for Spanish translation. IMHO to keep it simple the PR should only include changes in this language. I would like to propose a different PR for those typos, as it is possible more of them in other languages.

Reply:
In principle you are right I think.

However personally I would let this one slide, as it seems so more overhead to create a new PR for a single character fix. I just want to make sure we get the fix in :)

Btw good job spotting this issue in the first place @ovas04 . Eagle eyes! 🦅 👁️ :)

Reply:
Got it @spier ! I added other comments in the same way, so you could skip them. I will not add more reflections in that way. Cheers!